config/wireshark.mpb

```diff
@@ -73,7 +73,7 @@ feature(wireshark) {
 feature(wireshark_cmake) {
   avoids += wireshark
 
-  includes    += $(WIRESHARK_BUILD)
+  includes    += $(WIRESHARK_BUILD) $(WIRESHARK_SRC)/include
```

Comment:
Does the comment above need to be changed for this?

Reply:
Is `$(WIRESHARK_SRC)/include` only for wireshark_cmake or should it be a general wireshark setting?

Reply:
> Does the comment above need to be changed for this?

`WIRESHARK_SRC` is already required by the generic base project, so I don't think so.

> Is `$(WIRESHARK_SRC)/include` only for wireshark_cmake or should it be a general wireshark setting?

No. `wireshark_cmake` is only needed when using a CMake build directory and when you install Wireshark and would need to use `wireshark_cmake=0`, the headers from `$(WIRESHARK_SRC)/include` are in the same place as before.

Reply:
Right, I missed the part about it listing the "additional paths", so the one from above is still relevant for the CMake-built wireshark.